test/resources/example_file.txt

```diff
@@ -0,0 +1,7 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+Etiam eget ligula eu lectus lobortis condimentum.
+Aliquam nonummy auctor massa.
+Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.
+Nulla at risus.
+Quisque purus magna, auctor et, sagittis ac, posuere eu, lectus.
+Nam mattis, felis ut adipiscing.
```

Comment:
I think we need a new empty line here.
https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Nice catch